src/main/resources/languages/ko-KR.json

```diff
@@ -1,28 +1,28 @@
 {
   "messages": {
     "game": {
-      "address_bind": "🇺🇸Game Server started at \u001b[1m\u001b[33m%s:%s\u001b[0m",
+      "address_bind": "게임 서버가 \u001b[1m\u001b[33m%s:%s\u001b[0m에서 열렸습니다.",
       "port_bind": "게임 서버가 포트 %s에서 열렸습니다.",
       "connect": "클라이언트가 %s에서 연결됐습니다.",
       "disconnect": "클라이언트가 %s에서 연결이 끊겼습니다.",
       "game_update_error": "게임 업데이트 도중 오류가 발생했습니다.",
       "command_error": "명령어 오류:"
     },
     "dispatch": {
-      "address_bind": "🇺🇸[Dispatch] Dispatch server started at \u001b[1m\u001b[33m%s:%s\u001b[0m",
+      "address_bind": "[Dispatch] 디스패치 서버가 \u001b[1m\u001b[33m%s:%s\u001b[0m에서 열렸습니다.",
       "port_bind": "[Dispatch] 디스패치 서버가 포트 %s에서 열렸습니다.",
-      "request": "🇺🇸[Dispatch] Client %s %s request: %s",
+      "request": "[Dispatch] 클라이언트 %s %s 응답: %s",
       "keystore": {
         "general_error": "[Dispatch] 키스토어 로딩중 오류가 발생했습니다!",
         "password_error": "[Dispatch] 키스토어를 로딩할 수 없습니다. 기본 키스토어 암호로 시도 중...",
         "no_keystore_error": "[Dispatch] SSL인증서를 찾을 수 없습니다! HTTP서버로 되돌리는 중.",
         "default_password": "[Dispatch] 기본 키스토어 암호가 성공적으로 로딩됐습니다. config.json에서 암호를 123456으로 설정해 보십시오."
       },
       "authentication": {
-        "default_unable_to_verify": "🇺🇸[Authentication] Something called the verifyUser method which is unavailable in the default authentication handler."
+        "default_unable_to_verify": "[Authentication] Something called the verifyUser. 기본 인증 처리기에서 사용할 수 없는 매서드입니다."
```

Comment:
Don't want to do anything about "Something called the"?

Reply:
Oh, I didn't see that
I revised the other parts and commited them

Reply:
Thanks!